src/evt/evt.c

```diff
@@ -1341,15 +1354,15 @@ ApiStatus func_802C73B0(Evt* script) {
     return ApiStatus_DONE2;
 }
 
-s32 func_802C73B8(Evt* script) {
+ApiStatus func_802C73B8(Evt* script) {
```

Comment:
isInitiaCall arg

Reply:
these (this and the two above) don't have those args